lib/factory_factory_girl/configuration.rb

```diff
@@ -16,13 +16,13 @@ def initialize
       @rails_options = {}
     end
 
-    def match(rule, value: nil, function: nil)
-      raise "Need to give attribute or process" if                 value.nil? && function.nil?
+    def match(attributes, value: nil, function: nil)
```

Comment:
How about `pattern`?
We usually describe regular expression as a pattern.
[Reference](https://en.wikipedia.org/wiki/Regular_expression#Basic_concepts)